### PR TITLE
#5143 [QA] Add a new task for tests of UI flow

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5143)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/build.gradle [hunk 1]
```diff
@@ -106,6 +106,21 @@ task api {
   dependsOn setup
 }
 
+task it {
```
Comment (saivirtue) [inline ohara-manager/build.gradle:109]: @jackyoh 
I had added a new gradle task `it` for manager, please help me to add a new Jenkins command to trigger this one :)

File: ohara-manager/scripts/runTests.js [hunk 2]
```diff
@@ -150,7 +158,7 @@ const run = async (prod, apiRoot, serverPort = 5050, clientPort = 3000) => {
   } catch (err) {
     console.log(chalk.red(err.message));
   } finally {
-    await mergeReports('clientE2e');
+    await mergeReports(`client_${testMode}`);
```
Comment (saivirtue) [inline ohara-manager/scripts/runTests.js:161]: @jackyoh @chia7712 
I had changed the report file name from
- API : `clientApi`
- E2E: `clientE2e`
to
- API: `client_api`
- E2E: `client_e2e`
- IT: `client_it`

Does this change affect the reporting job of Jenkins?

Comment (saivirtue) [reply ohara-manager/scripts/runTests.js:161]: Never mind...It looks did break the rule 😅 
I will roll back the rule of report file name

Comment (jackyoh) [reply ohara-manager/scripts/runTests.js]: 做以上的修改 QA 的設定也要做修改，像是 report 的檔案名稱，要從 clientE2e.xml 改成 client_E2e.xml

Comment (saivirtue) [reply ohara-manager/scripts/runTests.js]: 感謝驗證~
這裡的邏輯我會改回原本的檔名：
`clientE2e.xml`, `clientApi.xml`, and `clientIt.xml`

File: ohara-manager/build.gradle (hunk 1 above)
Comment (jackyoh) [reply ohara-manager/build.gradle:109]: OK,  我再把這一段加到 jenkins 的設定

File: ohara-manager/build.gradle [hunk 3]
```diff
@@ -106,6 +106,21 @@ task api {
...
+  if(!project.hasProperty('skipManager')) {
+    doLast {
+      exec {
+        if(!project.hasProperty('ohara.manager.it.configurator')) throw new IllegalArgumentException("ohara.manager.it.configurator is required")
+        String configurator = project.getProperty('ohara.manager.it.configurator')
```
Comment (chia7712) [inline ohara-manager/build.gradle:114]: 是否加上註解說明IT需要使用fake mode?

或者另一種作法是將參數名稱換成```ohara.manager.fake.configurator```

**Comment by jackyoh** [comment]
retry manager-it

File: ohara-manager/build.gradle (hunk 3 above)
Comment (saivirtue) [reply ohara-manager/build.gradle:114]: > 是否加上註解說明IT需要使用fake mode?

Sure

**Comment by saivirtue** [comment]
retry

**Comment by jackyoh** [comment]
retry manager-it

**Comment by jackyoh** [comment]
retry manager-it

**Comment by saivirtue** [comment]
retry manager-it

**Comment by saivirtue** [comment]
Big Thanks @jackyoh !

**Pull request merged**